status-reporter/status_reporter/postgresql_reporter.py

```diff
@@ -45,11 +45,12 @@ def _collect_status(self):
         return status, services
 
     def _get_patroni_status(self):
-        config = get_cloudify_config(logger)
-        if not config:
+        extra_config = self._config.get('extra_config', {})
```

Comment:
There is not const for 'extra_config'?

Reply:
There is now :)
There was in the manager-install repo